m4_sessions/1801-inning/Group_1.md

```diff
@@ -1,13 +1,19 @@
 Group 1
 
+Group Members: Liam Barstad, Joan Harrington, Aaron Schreck, Iza Jimenez
 
-Group Members:
-
-Name of Session:
+Name of Session: Ethics in Ai
 
 Session Description (Why should students come to this session):
+As tech developers well have ethical responsibilies.
+Discuss singularity, Moores Law inevitability.
+Practices like machine learning will become more common place, what are the ethical ramifications?
 
 Session Outcomes (What will students learn/leave with?):
+Different views on Ai, possible ramifications both short and long term.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+10min Ice breakers/intros
+15min Current perspectives of Ai
```

Comment:
Have you started a channel for resources?  Can you add me?  

Ideal to gather a bunch of resources on perspectives in the channel.  Then week 3 you can narrow this down and decide what are the questions that are most controversial.  You might need to play "devil's advocate" so prepare talking points for at least two perspectives.  You want to push students to think deeply about this concept, not just agree that something is good or bad.  Think about how AI will impact your life in the future.  I love my Alexa and I also know that might mean I'm being recorded all the time.  So how do I feel about that?